Matrix.py

```diff
@@ -108,9 +106,14 @@ def __truediv__(self, other):
         if type(other) is float or int:  # the program doesnt support division by other Matrix
             return self.scalar_mul(1 / other)
         else:
-            raise SyntaxError("only  scalar division")
+            raise SyntaxError("only scalar division")
 
     def __mul__(self, other):
+        """mul function decides which multiply method is needed for each call
+        args: self.data(matrix)- the first factor
+         other(int/float/matrix)- the second factor
+         no returns
+         exceptions: SyntaxError if other isn't a matrix or a scalar"""
```

Comment:
You don't have to write doc for built-in function

Reply:
but its not a "common" __mul__ i used it for different purpose. (i used __mul__  for the user, he only need to type * without consider if its a scalar or matrices multiplication.

Reply:
Like I said you don't have to.
You can leave the doc, but make it according to conventions